test/unit/account/search_test.rb

```diff
@@ -111,6 +113,22 @@ class Account::SearchTest < ActiveSupport::TestCase
     assert_equal 0, result.size
   end
 
+  test 'search user_key without keyword with many records is always indexes and found' do
+    service = FactoryBot.create(:simple_service)
+    buyer = FactoryBot.create(:simple_buyer)
+    FactoryBot.create_list(:application_plan, 70, issuer: service).each do |plan|
```

Comment:
Do you know how long does that takes?

Reply:
Yeah, about 14 seconds in my laptop, but what do you propose otherwise?

Reply:
From Hery:
> Not doing the test

Reply:
After the conversation with Hery: It is not worth having it tested if it will take 14 seconds. So I will remove it.